### DISCUSSION TITLE
What are the guidelines to run clustered quartz cron jobs in Kubernetes

### DISCUSSION THREAD

**Discussion by sidduNT** [quartznet/quartznet#1742] [Q&A]
Currently, I'm using Quartz.NET cron jobs as hosted service(multiple instances) in IIS.
What are the guidelines to run clustered quartz cron jobs in Kubernetes?

**Comment by lahma** [comment]
I haven't done any K8S development, but I would assume that same ideas apply. You need a database to coordinate locking and clustering. I would assume that dedicated pod would work better than hosting inside IIS anyway, but I can always be wrong.
